package.json

```diff
@@ -1,74 +1,65 @@
 {
-  "name": "magazine",
+  "name": "leetcode-spa",
+  "private": true,
+  "version": "0.0.0",
   "type": "module",
-  "version": "0.0.1",
+  "license": "MIT",
   "scripts": {
-    "dev": "astro dev",
-    "build": "astro build",
-    "preview": "astro preview",
-    "astro": "astro",
-    "lint": "eslint . --ext .js,.mjs,.cjs,.ts,.tsx,.jsx,.astro",
-    "lint:fix": "npm run lint -- --fix",
-    "format": "prettier . --write",
-    "format:check": "prettier . --check",
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --report-unused-disable-directives",
+    "lint:fix": "eslint . --fix",
+    "preview": "vite preview",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "test": "vitest",
     "test:ui": "vitest --ui",
-    "test:run": "vitest run",
     "test:coverage": "vitest run --coverage",
     "test:watch": "vitest --watch"
   },
-  "dependencies": {
-    "@astrojs/react": "4.3.1",
-    "@emotion/cache": "11.14.0",
-    "@emotion/react": "11.14.0",
-    "@emotion/server": "11.11.0",
-    "@emotion/styled": "11.14.1",
-    "@mui/icons-material": "7.3.4",
-    "@mui/material": "7.3.4",
-    "astro": "5.13.9",
-    "react": "19.1.1",
-    "react-dom": "19.1.1"
-  },
   "engines": {
     "node": ">=20.3.0",
     "yarn": ">=1.22.21 <=1.22.22"
   },
+  "dependencies": {
```

Comment:
The repository still contains Astro-related configuration files (e.g. astro.config.mjs, .astro/*) but `astro` and `@astrojs/react` were removed from dependencies. This is inconsistent: you either migrate away from Astro (remove config) or keep Astro deps/config consistent.

**Sugerencia:**
```typescript
Either restore Astro dependencies (if the project uses Astro) or remove astro.config.mjs and `.astro` references and update tsconfig accordingly. Keep configuration and dependencies aligned. Example: if moving to Vite, remove astro.config.mjs and `'.astro/types.d.ts'` from tsconfig include.
```